public/javascripts/Admin/src/Admin.GSVLabelView.js

```diff
@@ -188,24 +188,45 @@ function AdminGSVLabelView(admin, source) {
         }
         self.prevAction = null;
 
-        self.agreeButton.click(function() {
-            if (self.prevAction !== "Agree") {
-                _disableValidationButtons();
-                _validateLabel("Agree");
-            }
-        });
-        self.disagreeButton.click(function() {
-            if (self.prevAction !== "Disagree") {
-                _disableValidationButtons();
-                _validateLabel("Disagree");
-            }
-        });
-        self.unsureButton.click(function() {
-            if (self.prevAction !== "Unsure") {
-                _disableValidationButtons();
-                _validateLabel("Unsure");
-            }
-        });
+        self.commentButton = self.modal.find("#comment-button");
+        self.commentTextArea = self.modal.find("#comment-textarea");
+
+        // Hide validation buttons and comment submission for images labeled by current user
+        if (self.source === "UserMap") {
+            self.modal.find('#validation-input-holder h3').hide();
```

Comment:
Instead of hiding all those pieces individually, why not just hide the `#validation-intput-holder`? I think that it would both be simpler _and_ look nicer (there's some weird white space between the image and the table now)!